modules/campaignion_flexible_form/campaignion_flexible_form.features.field_instance.inc

```diff
@@ -104,7 +104,7 @@ function campaignion_flexible_form_field_default_field_instances() {
         'summary_rows' => 5,
       ),
       'type' => 'text_textarea_with_summary',
-      'weight' => 5,
+      'weight' => 15,
```

Comment:
I don’t see any reason for the weights in the content group to change? Would prefer to keep the diff as minimal as possible. If I haven’t overlooked something, only group weights and the pgbar field weight need to be updated.

Reply:
I _think_ I double checked now which weights should change and which shouldn't but happy if take a close look again :eyes:

Reply:
Thanks!

For campaigns, I think the new field should be in `group_only_published` as it only makes sense when the campaign has an actual page → then the description field doesn’t change.

The change in `node-email_to_target-field_no_target_message` and `node-email_to_target-field_email_to_target_options` looks justified.

Reply:
cooli, have exported it again :)